pkg/util/util.go

```diff
@@ -0,0 +1,29 @@
+package util
+
+import (
+	"os"
+	"path/filepath"
+)
+
+// ToFileName converts a name to be alphanumeric plus underscores.
+func ToFileName(name string) string {
+	for i := 0; i < len(name); i++ {
+		if (name[i] < 'a' || name[i] > 'z') && (name[i] < 'A' || name[i] > 'Z') && (name[i] < '0' || name[i] > '9') {
+			name = name[:i] + "_" + name[i+1:]
+		}
+	}
+	return name
```

Comment:
The if-statement hurts my eyes. Not sure if this is easier for you to read. This will leave "unicode letters" (like `世界`), but that's probably OK?
```suggestion
	return strings.Map(func(c rune) rune {
		if !unicode.IsLetter(c) && !unicode.IsDigit(c) {
			return '_'
		}
		return c
	}, name)
```

Reply:
Nice, I didn't know about the unicode package. I'll use this